### PR TITLE
docs(blog): add screen reader improvement post

### PR DESCRIPTION
**Pull request by chasestarr**

### CODE REVIEW

File: documentation-site/pages/blog/screen-reader-improvements/index.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Is my application accessible when using Base Web?
+
+If you're using Base Web rest assured that a significant portion of your application will be fine, but accessible components are not an entirely accessible application. You absolutely need to audit your application to be sure all every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.
```
Comment (gergelyke) [inline documentation-site/pages/blog/screen-reader-improvements/index.mdx]:
```suggestion
If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.
```

**Comment by gergelyke** [comment]
nit: can we illustrate any of the paragraphs with a few pictures / maybe embed some of the videos?

File: documentation-site/pages/blog/screen-reader-improvements/index.mdx (hunk 1 above)
Comment (tajo) [inline documentation-site/pages/blog/screen-reader-improvements/index.mdx]: https://w3c.github.io/aria-practices/#Listbox -> https://w3c.github.io/aria-practices/

File: documentation-site/pages/blog/screen-reader-improvements/index.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Is my application accessible when using Base Web?
+
+If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.
```
Comment (chasestarr) [inline documentation-site/pages/blog/screen-reader-improvements/index.mdx]:
```suggestion
If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices) document.
```

**Pull request merged**